Source/Core/VideoCommon/PostProcessing.cpp

```diff
@@ -209,39 +365,264 @@ void PostProcessingShaderConfiguration::LoadOptions(const std::string& code)
 				}
 			}
 		}
+		option.m_default_bool_value = option.m_bool_value;
+		option.m_default_float_values = option.m_float_values;
+		option.m_default_integer_values = option.m_integer_values;
 		m_options[option.m_option_name] = option;
 	}
+
+	for (const auto& it : pass_strings)
+	{
+		RenderPass pass;
+		pass.output_scale = 1.0f;
+
+		for (const auto& option : it.m_options)
```

Comment:
This needs to be broken out into constituent functions, this is way too monolithic to track behaviour.